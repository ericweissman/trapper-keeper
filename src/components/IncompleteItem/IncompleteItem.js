import React from 'react'
import unchecked from '../../images/unchecked.svg'
import PropTypes from 'prop-types';

export const IncompleteItem = (props) => {
  return (
    <div className='incomplete'>
      <img src={unchecked}/>
      <p>{props.description}</p>
    </div>
  )
}

export default IncompleteItem

IncompleteItem.propTypes = {
  description: PropTypes.string,
}