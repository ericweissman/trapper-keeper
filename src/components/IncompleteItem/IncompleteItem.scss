.incomplete {
  display: flex;
  align-items: center;
  padding: 3px 0;
  p {
    padding-left: 10px;
  }
}