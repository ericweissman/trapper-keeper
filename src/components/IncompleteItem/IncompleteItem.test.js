import React from 'react'
import { shallow } from 'enzyme'
import IncompleteItem from './IncompleteItem'

describe('IncompleteItem', () => {
  let wrapper
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<IncompleteItem />)
    expect(wrapper).toMatchSnapshot()
  })
})