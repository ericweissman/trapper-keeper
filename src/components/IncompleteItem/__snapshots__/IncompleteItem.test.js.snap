// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IncompleteItem should have match the correct snapshot 1`] = `
<div
  className="incomplete"
>
  <img
    src="unchecked.svg"
  />
  <p />
</div>
`;
